addons/website_slides/controllers/main.py

```diff
@@ -434,9 +434,28 @@ def slides_channel_all_values(self, slide_category=None, slug_tags=None, my=Fals
         )
         search = post.get('search')
         order = self._channel_order_by_criterion.get(post.get('sorting'))
-        search_count, details, fuzzy_search_term = request.website._search_with_fuzzy("slide_channels_only", search,
-            limit=1000, order=order, options=options)
-        channels = details[0].get('results', request.env['slide.channel'])
+        channel_search_count, channel_details, channel_fuzzy_search_term = request.website._search_with_fuzzy(
+            "slide_channels_only", search, limit=1000, order=order, options=options)
+        channels = channel_details[0].get('results', request.env['slide.channel'])
+
+        # Don't show contents because tags are related to courses
+        if options['tag'] is None:
+            slide_search_count, slide_details, slide_fuzzy_search_term = request.website._search_with_fuzzy("slides_only",
+                channel_fuzzy_search_term or search, limit=1000, order="" if order is None else order, options=options)
```

Comment:
If `channel_fuzzy_search_term` exist here, there will be no search on the real term the user typed for `slides.slide`. So results could exist but won't be displayed because the search was done on the fuzzy_term. Not sure that's what we want ^^

Maybe do `_search_with_fuzzy("slides", search, limit=1000, order=order, options=options)` and then display the slides only if there is no tags ?